### ISSUE TITLE
safari on ios auto zooms text inputs

### ISSUE THREAD

**Issue by sudhakar-selva** [razorpay/blade#1476]
we are using fontsize 14px for text input(we may have to check other inputs), this would cause safari on ios to zoom-in the input when user try to type something, this could lead to bad UX. one way to solve this is to set fontsize to 16px not sure if blade allows it

[Slack](https://razorpay.slack.com/archives/CMQ3RBHEU/p1690355598721839)
Other Sources: [Link](https://twitter.com/joshwcomeau/status/1379782931116351490?s=12)

**Comment by kamleshchandnani** [comment]
@sudhakar-selva did you try adding the following meta tag to your index.html? 
`<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">`

this is what we generally use to tell browsers not to auto-scale since we have interfaces designed to work across different screen sizes.

**Comment by kamleshchandnani** [comment]
From internal Slack discussions:

In iOS 10, [Apple changed its handling](https://webkit.org/blog/7367/new-interaction-behaviors-in-ios-10/) of user-scalable, min-scale, and max-scale declarations. Their changes removed the effect on pinch-zooming, making it possible to pinch-zoom regardless of the meta element's suggestions.

android might have an issue and I think we can conditionally add meta tags. because changing the font size is also a temporary fix. if the user has base font size default to 18px then even if we bump to 16px that zoom might again happen.

We can add this meta attribute conditionally for iOS
```js
if(navigator.userAgent.indexOf('iPhone') > -1 )
{
    document
      .querySelector("[name=viewport]")
      .setAttribute("content","width=device-width, initial-scale=1, maximum-scale=1");
}
```

so basically Blade has nothing to do here. This shall be handled at the application level. Closing this in favor of the above recommendation
